config/client-boot.js

```diff
@@ -0,0 +1,27 @@
+import { h, render } from 'preact';
+
+if (process.env.NODE_ENV === 'development') {
+	// enable preact devtools
+	require('preact/debug');
+}
+else if (process.env.ADD_SW && 'serviceWorker' in navigator && location.protocol === 'https:') {
+	// eslint-disable-next-line no-undef
+	navigator.serviceWorker.register(__webpack_public_path__ + 'sw.js');
+}
+
+const interopDefault = m => m && m.default ? m.default : m;
```

Comment:
Expand this and add a couple of comments, the minifier will hopefully make it as small as it is now :D

Reply:
- [ ] comment